utils/guardrails/guard.py

```diff
@@ -22,12 +22,10 @@ class Guard:
 
     def __init__(
         self,
-        api: str = 'sambaverse',
+        api: str = 'sambastudio',
```

Comment:
Just to confirm, shouldn't the default here be sncloud?

Reply:
no llamaguard is not available yet in sncloud